packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts

```diff
@@ -39,7 +38,7 @@ export function setWorkspaceOrgTypeWithOrgType(orgType: OrgType) {
 export async function setupWorkspaceOrgType(defaultUsernameOrAlias?: string) {
   try {
     setHasDefaultUsername(!!defaultUsernameOrAlias);
-    const orgType = await getWorkspaceOrgType(defaultUsernameOrAlias);
+    const orgType = await getWorkspaceOrgType();
```

Comment:
On line 77, there is:
```
'sfdx:default_username_has_no_change_tracking',
```

Should it be
```
sfdx:default_username_has_no_change_tracking || !sfdx:is_scratch_org
```
(like that in packages/salesforcedx-vscode-core/package.json?)

I don't know if it should or not, but I saw this here, and saw that the other instances of `sfdx:default_username_has_no_change_tracking` was changed to also check for !is_scratch_org and was wondering if this needs to be updated as well.

Reply:
The change in package.json to include "!is_scratch_org" is so that the deploy and retrieve commands are still exposed for src-trk sandboxes (but not yet for scratch orgs).  

Here the code is simply setting whether or not the current ordered has change/source tracking.  So !is_scratch_org is not needed here. 

And the is_scratch_org context var is being set now in workspaceContext.